openshift-ci/deploy_ingress_node_firewall.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/bash
+
+ingress_node_firewall_dir="$(dirname $(readlink -f $0))"
+source ${ingress_node_firewall_dir}/common.sh
+
+INGRESS_NODE_FIREWALL_IMAGE_BASE=${INGRESS_NODE_FIREWALL_IMAGE_BASE:-$(echo "${OPENSHIFT_RELEASE_IMAGE}" | sed -e 's/release/stable/g' | sed -e 's/@.*$//g')}
+INGRESS_NODE_FIREWALL_IMAGE_TAG=${INGRESS_NODE_FIREWALL_IMAGE_TAG:-"ingress-node-firewall"}
+INGRESS_NODE_FIREWALL_DAEMON_IMAGE_TAG=${INGRESS_NODE_FIREWALL_DAEMON_IMAGE_TAG:-"ingress-node-firewall-daemon"}
+export NAMESPACE=${NAMESPACE:-"openshift-ingress-node-firewall"}
+
+rm -f ingress-node-firewall-operator-deploy/bundle.Dockerfile
+rm -rf ingress-node-firewall-operator-deploy/bundle
+
+cp ../bundle.Dockerfile ingress-node-firewall-operator-deploy 
+cp -r ../bundle/ ingress-node-firewall-operator-deploy/bundle 
+
+cd ingress-node-firewall-operator-deploy
+
+ESCAPED_OPERATOR_IMAGE=$(printf '%s\n' "${INGRESS_NODE_FIREWALL_IMAGE_BASE}:${INGRESS_NODE_FIREWALL_IMAGE_TAG}" | sed -e 's/[]\/$*.^[]/\\&/g');
+find . -type f -name "*clusterserviceversion*.yaml" -exec sed -i 's/quay.io\/mmahmoud\/controller:.*$/'"$ESCAPED_OPERATOR_IMAGE"'/g' {} +
```

Comment:
can we just replace the image instead of matching against my userID ?

Reply:
i was thinking pointing to your image is temp and we are going to replace all that with an image in quay.io/openshift/ingress-node-firewall, no?